usaspending_api/transactions/delta_models/transaction_search.py

```diff
@@ -390,6 +390,8 @@
     "veterinary_hospital": {"delta": "BOOLEAN", "postgres": "BOOLEAN", "gold": True},
     "woman_owned_business": {"delta": "BOOLEAN", "postgres": "BOOLEAN", "gold": True},
     "women_owned_small_business": {"delta": "BOOLEAN", "postgres": "BOOLEAN", "gold": True},
+    "program_activity_names": {"delta": "ARRAY<STRING>", "postgres": "TEXT[]", "gold": False},
```

Comment:
I don't remember the reason for the `"gold"` key here. Is it okay that this is `False`?

Reply:
Just as a note, the tech debt to remove the `gold` keys would be:
- Remove the `gold` keys from `delta_models/award_search.py`.
- Remove `if not v["gold"]` from the `AWARD_SEARCH_POSTGRES_COLUMNS` dictionary in `delta_models/award_search.py`
- Remove the `award_search_gold` dictionary from the `load_query_to_delta.py` file.

Mostly I wanted to note this so I don't forget it, but it's a good reminder that this field doesn't mean anything anymore.